### DISCUSSION TITLE
Redirect STDOUT (print statements etc) to text box

### DISCUSSION THREAD

**Discussion by KingKDot** [Textualize/textual#3919] [Q&A] [answered]
Is it possible to redirect all my print statements and stdout into a ScrollableContainer and have it automatically update with the Reactive class?

**Comment by TomJGooding** [comment, marked as answer]
You could use the [`Log`](https://textual.textualize.io/widgets/log/) widget with the [`begin_capture_print`](https://textual.textualize.io/api/widget/#textual.widget.Widget.begin_capture_print) method.

If you search the discussions you'll find some examples of this, e.g. https://github.com/Textualize/textual/discussions/3505#discussioncomment-7312952.

**Reply by KingKDot** [reply]
That is exactly what I was looking for. Thank you for your help.
